src/System.Net.Requests/pkg/System.Net.Requests.pkgproj

```diff
@@ -2,14 +2,8 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\3.9.0\System.Net.Requests.depproj">
-      <SupportedFramework>wp8</SupportedFramework>
-    </ProjectReference>
-    <ProjectReference Include="..\ref\4.0.0\System.Net.Requests.depproj">
-      <SupportedFramework>net45;netcore45;wpa81</SupportedFramework>
-    </ProjectReference>
     <ProjectReference Include="..\ref\System.Net.Requests.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
>Currently the newer version is targeting netstandard17 which isn't supported on net46 so we cannot expose the new APIs there in a netstandard fashion

I understand that IF we move this contract to netstandard17, we need to express the .NET Framework as net463 and not net46.

So, my real question, then, is why is this revised contract moving at all to netstandard17? The added APIs for System.Net.Request are already in the .NET Framework.  So, the revised contract (4.1) could stay in netstandard13.